app-server/app/src/main/scala/user/User.scala

```diff
@@ -20,14 +47,37 @@ object UserService {
 
     database.db.run {
       Users.filter(_.id === id).result.headOption
+    } map {
+      userOption => userOption match {
+        case Some(user) => new UsersRowApi(user)
+        case _ => {
+          // catch this to return 404
+          throw new Exception("404 Resource not found")
+        }
+      }
     }
   }
 
-  def createUser(user:UsersRow)(implicit database:Database, ec:ExecutionContext) = {
+  def createUser(user:UsersRowCreate)(implicit database:Database, ec:ExecutionContext) = {
```

Comment:
I think we should add return types to the functions in this file, makes it easier to tell what's going on here